src/duckling/migration/map-migration.ts

```diff
@@ -3,7 +3,7 @@ import {MapVersion, compareVersions, versionCompareFunction, VersionCompatibilit
  * Represents a single migration that needs to be run when updating to the specified version.
  */
 export interface MapMigration {
-    type : "code" | "editor-version"; // | "editor-built-in",
+    type : "code" | "editor-version" | "existing-code", // "editor-version" is really "engine-version"
```

Comment:
Fixed. 

Tested for code migrations, not tested for existing-code/editor-version migrations